src/models/user.ts

```diff
@@ -0,0 +1,19 @@
+import * as fs from 'fs'
```

Comment:
it's not correct to import all functions from library
its info from Python, but I guess it works also in NodeJS 
https://stackoverflow.com/questions/2386714/why-is-import-bad :)
https://stackoverflow.com/questions/36795819/when-should-i-use-curly-braces-for-es6-import/36796281

Reply:
Done